server/lib/packets/login.go

```diff
@@ -70,11 +69,10 @@ func ProcessLogin(from Source, msgid uint32, req *proto.LoginRequest, t Transpor
 	//set device information
 	device_type := req.DeviceType
 	device_id := req.DeviceId
-	idstr := a.StringId()
 	if device_id == nil {
 		//TODO: get unique identifier of browser and use it as device_id.
 		//(especially, if user uses mobile browser, how we identify it as same mobile device)
-		tmp_device_id := "browser:" + idstr
+		tmp_device_id := "browser:" + a.StringId()
```

Comment:
__[gometalinter]__
* [golint] don't use underscores in Go names; var tmp_device_id should be tmpDeviceID